chia/_tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -98,7 +107,10 @@ async def test_nft_wallet_creation_automatically(wallet_environments: WalletTest
         wallet_node_0.wallet_state_manager, wallet_0, name="NFT WALLET 1"
     )
     metadata = Program.to(
-        [("u", ["https://www.chia.net/img/branding/chia-logo.svg"]), ("h", "0xD4584AD463139FA8C0D9F68F4B59F185")]
+        [
+            ("u", ["https://www.chia.net/img/branding/chia-logo.svg"]),
+            ("h", "0xD4584AD463139FA8C0D9F68F4B59F185D4584AD463139FA8C0D9F68F4B59F185"),
```

Comment:
is there a case where the shorter value is appropriate to use?  i'm guessing this changed because the new interface is requiring a 'full address'.